pyproject.toml

```diff
@@ -80,6 +80,9 @@ ldp = [
 local = [
     "sentence-transformers",
 ]
+qdrant = [
+    "qdrant-client",
```

Comment:
Can you add this to the `mypy` section of `.pre-commit-config.yaml`

Reply:
Done with https://github.com/Future-House/paper-qa/pull/730/commits/d15f3f38160399bae3af07c3d28ce9743ab50e8c.